ramls/voucher.raml

```diff
@@ -0,0 +1,53 @@
+#%RAML 1.0
+
+title: Vouchers
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://github.com/folio-org/mod-invoice-storage
+
+documentation:
+  - title: Vouchers CRUD API
+    content: This documents the API calls that can be made to manage vouchers
```

Comment:
Can you please add a note that this API is intended for internal use only?  See https://github.com/folio-org/mod-orders-storage/blob/2e36263a657ccabc2d7eb4b96850f166ea3b20e5/ramls/orders.raml#L8

You can leave the suggestion part out since we don't yet expose the ability to create vouchers manually - only upon invoice approval

Reply:
Done